internal/controlplane/handlers_repositories_test.go

```diff
@@ -305,16 +307,8 @@ func (*StubGitHub) CreateComment(context.Context, string, string, int, string) e
 }
 
 // CreateHook implements v1.GitHub.
-func (s *StubGitHub) CreateHook(_ context.Context, owner string, repo string, hook *github.Hook) (*github.Hook, error) {
```

Comment:
These mock methods are no longer needed: the test relies on the stub of `WebhookManager` instead

Reply:
Yeah, `GetRepository` is still used.